automation/Dockerfile.fedora

```diff
@@ -0,0 +1,34 @@
+# This Dockerfile is based on the recommendations provided in the
+# Centos official repository (https://hub.docker.com/_/centos/).
+# It enables systemd to be operational.
+FROM fedora
+ENV container docker
+RUN (cd /lib/systemd/system/sysinit.target.wants/; for i in *; do [ $i == \
```

Comment:
If this is needed for both CentOS and Fedora, it might make sense to move this into a script